cas9-AA-subst.py

```diff
@@ -0,0 +1,51 @@
+#///////////////////////**READ ME FIRST**//////////////////////////////////
+# Point of this script is to substitute amino acids into a given pose 
+#  protein structure in pyrosetta.
+# Please change dropbox folder location accordingly + make sure that the 
+#  script's current location is the root of PyRosetta (need to import stuff)
+#//////////////////////////////////////////////////////////////////////////
+## Raw file date modified: July 15th/2014
+
+## importing modules required for script
+import os
+from rosetta import *
+## difference between resfiles and mutate_residue is that the former takes into account 
+##  rotamers (aka conformers) whereas the latter doesn't (for our case mutate_residue is sufficient)
+##  refer to link for further documentation and/or info:
+##  http://graylab.jhu.edu/pyrosetta/downloads/documentation/Workshop6_PyRosetta_Packing_Design.pdf
+from toolbox import generate_resfile_from_pdb # generate mutations using resfiles
+from toolbox import mutate_residue # generate mutations using mutate_residue  
+
+## changing directory to where PDB's are located (aka where PDB files are located )
+os.chdir("~\Dropbox\Waterloo-iGEM-2015") #alter to your specific dropbox path
+os.chdir("\Math Modelling\cas9_modification") ##where the WT cas9 should be located???
+## not sure if completely correct???? add changes if not.
+
+## initializing rosetta:
+rosetta.init()
+
+# import cleaning module for PDB to be usable
+from toolbox import cleanATOM
```

Comment:
Also Kim, I think it would be clearer if you wrapped the things going on in your script into a function or two (and outline the input / output / pdb file writing in a docstring), then had a main segment at the bottom which allowed for user input.